uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -29,31 +36,51 @@
 M_PATH = "uaclient.entitlements.livepatch."  # mock path
 M_BASE_PATH = "uaclient.entitlements.base.UAEntitlement."
 
+DEFAULT_AFFORDANCES = {
+    "architectures": ["x86_64"],
+    "minKernelVersion": "4.4",
+    "kernelFlavors": ["generic", "lowlatency"],
+    "tier": "stable",
+}
+
 
 @pytest.fixture
-def entitlement(entitlement_factory):
-    affordances = {
-        "architectures": ["x86_64"],
-        "minKernelVersion": "4.4",
-        "kernelFlavors": ["generic", "lowlatency"],
-        "tier": "stable",
-    }
-    directives = {"caCerts": "", "remoteServer": "https://alt.livepatch.com"}
-    return entitlement_factory(
-        LivepatchEntitlement, affordances=affordances, directives=directives
-    )
+def livepatch_entitlement_factory(entitlement_factory):
```

Comment:
When writing the fips equivalent to this, I realised we could make this simpler with `functools.partial` too:

```diff
diff --git a/uaclient/entitlements/tests/test_livepatch.py b/uaclient/entitlements/tests/test_livepatch.py
index 0e36780..26872c7 100644
--- a/uaclient/entitlements/tests/test_livepatch.py
+++ b/uaclient/entitlements/tests/test_livepatch.py
@@ -3,6 +3,7 @@
 import copy
 import logging
 import mock
+from functools import partial
 from types import MappingProxyType
 
 try:
@@ -46,26 +47,13 @@ DEFAULT_AFFORDANCES = {
 
 @pytest.fixture
 def livepatch_entitlement_factory(entitlement_factory):
-    def factory_func(
-        entitled: bool = True,
-        affordances: "Dict[str, Any]" = None,
-        suites: "List[str]" = None,
-    ):
-        if not affordances:
-            affordances = DEFAULT_AFFORDANCES
-        directives = {
-            "caCerts": "",
-            "remoteServer": "https://alt.livepatch.com",
-        }
-        return entitlement_factory(
-            LivepatchEntitlement,
-            affordances=affordances,
-            directives=directives,
-            entitled=entitled,
-            suites=suites,
-        )
-
-    return factory_func
+    directives = {"caCerts": "", "remoteServer": "https://alt.livepatch.com"}
+    return partial(
+        entitlement_factory,
+        LivepatchEntitlement,
+        affordances=DEFAULT_AFFORDANCES,
+        directives=directives,
+    )
 
 
 @pytest.fixture
```